test/xspec-prefix-conflict-import.xspec

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<t:description
+	xmlns:t="http://www.jenitennison.com/xslt/xspec"
+	stylesheet="xspec-xmlns.xsl"
+	xslt-version="2.0">
+
+	<t:import href="xspec-xmlns-imported.xspec"/>
+
+</t:description>
```

Comment:
It's great that this `.xspec` file surely reproduces the problem described in https://github.com/xspec/xspec/pull/640#discussion_r333697769.
But I wonder why this file is called `xspec-prefix-conflict-import.xspec`. The other `xspec-prefix-conflict*.xspec` files try to use `x:` prefix for their own namespaces and see if XSpec doesn't conflict with it. This `xspec-prefix-conflict-import.xspec` file seem to try a different thing.

Reply:
You're right, it's a little different. Here, the point is that the parent file and the imported file use different prefixes for the XSpec namespace. Do you think `xspec-prefix-parent-vs-import.xspec` would be a better filename? It's kind of long, so feel free to suggest something better.

Reply:
> Here, the point is that the parent file and the imported file use different prefixes for the XSpec namespace.

It would be nice to write the point into this file as a comment.

> Do you think `xspec-prefix-parent-vs-import.xspec` would be a better filename? It's kind of long, so feel free to suggest something better.

Yes, I think it's better or less confusing at least. I tried to compose a shorter name, only to fail.

Reply:
Thanks. I renamed the file and included a comment, in acab3d491188b8ba22921e61c7d6a006f4eacfda.